src/cryptonote_protocol/cryptonote_protocol_handler.inl

```diff
@@ -284,10 +280,10 @@ namespace cryptonote
 	int64_t max_block_height = max(static_cast<int64_t>(hshd.current_height),static_cast<int64_t>(m_core.get_current_blockchain_height()));
 	int64_t last_block_v1 = 1009826;
 	int64_t diff_v2 = max_block_height > last_block_v1 ? min(abs(diff), max_block_height - last_block_v1) : 0;
-    LOG_PRINT_CCONTEXT_YELLOW("Sync data returned a new top block candidate: " << m_core.get_current_blockchain_height() << " -> " << hshd.current_height
+    MCLOG(is_inital ? el::Level::Info : el::Level::Debug, "global", context <<  "Sync data returned a new top block candidate: " << m_core.get_current_blockchain_height() << " -> " << hshd.current_height
```

Comment:
Could be. But:

- this is fixed somewhere else by Gingeropolous IIRC, this branch isn't yet rebased
- most of your comments are bikeshedding anyway
- Open new bugs for different things, we don't want irrelevant stuff in unrelated bugs